docs/release-info/release-notes.md

```diff
@@ -57,6 +57,125 @@ For tips about how to write a good release note, see [Release notes](https://git
+  "header": true,
+  "typesHeader": true,
+  "sqlTypesHeader": true
+}'
+```
+
+
+
+This improvement also works for INSERT and REPLACE queries using the MSQ task engine. Note that JDBC isn't supported.
+
+#### Improved HTTP endpoints
+
+You can now use raw SQL in the HTTP body for `/druid/v2/sql` endpoints. You can set `Content-Type` to `text/plain` instead of `application/json`, so you can provide raw text that isn't escaped. 
+
+
+ [#17937](https://github.com/apache/druid/pull/17937)
+
+### Cloning Historicals
+
+You can now configure clones for Historicals using the dynamic Coordinator configuration `cloneServers`. Cloned Historicals are useful for situations such as rolling updates where you want to launch a new Historical as a replacement for an existing one.
+
+Set the config to a map from the target Historical server to the source Historical:
+
+```
+  "cloneServers": {"historicalClone":"historicalOriginal"}
+```
+
+The clone doesn't participate in regular segment assignment or balancing. Instead, the Coordinator mirrors any segment assignment made to the original Historical onto the clone, so that the clone becomes an exact copy of the source. Segments on the clone Historical do not count towards replica counts either. If the original Historical disappears, the clone remains in the last known state of the source server until removed from the `cloneServers` config.
+
+When you query your data using the native query engine, you can prefer (`preferClones`), exclude (`excludeClones`), or include (`includeClones`) clones by setting the query context parameter `cloneQueryMode`. By default, clones are excluded.
+
+As part of this change, new Coordinator APIs are available. For more information, see [Coordinator APIs for clones](#coordinator-apis-for-clones).
+
+[#17863](https://github.com/apache/druid/pull/17863) [#17899](https://github.com/apache/druid/pull/17899) [#17956](https://github.com/apache/druid/pull/17956) 
+### Embedded kill tasks on the Overlord (Experimental)
+
+You can now run kill tasks directly on the Overlord itself. Embedded kill tasks provide several benefits; they:
+
+- Kill segments as soon as they're eligible 
+- Don't take up tasks slot
+- finish faster since they use optimized metadata queries and don't launch a new JVM
+- Kill a small number of segments per task, ensuring locks on an interval aren't held for too long
+- Skip locked intervals to avoid head-of-line blocking
+- Require minimal configuration
+- Can keep up with a large number of unused segments in the cluster
+
+This feature is controlled by the following configs:
+
+- `druid.manager.segments.killUnused.enabled` - Whether the feature is enabled or not
+- `druid.manager.segments.killUnused.bufferPeriod` - The amount of time that a segment must be unused before it is able to be permanently removed from metadata and deep storage. This can serve as a buffer period to prevent data loss if data ends up being needed after being marked unused.
```

Comment:
```suggestion
- `druid.manager.segments.killUnused.enabled` - Whether the feature is enabled or not (Defaults to `false`)
- `druid.manager.segments.killUnused.bufferPeriod` - The amount of time that a segment must be unused before it is able to be permanently removed from metadata and deep storage. This can serve as a buffer period to prevent data loss if data ends up being needed after being marked unused (Defaults to `P30D`)
```